docs/_documentations/troubleshooting.md

```diff
@@ -827,28 +945,21 @@ If you can run `docker` from the command line, but Codewind still fails to find
 3. Open the terminal you ran `docker` from and run `code`.
 4. Now, the new VS Code instance shares the terminal's `PATH`. Start Codewind again.
 
-<!--
-Action/Topic: Collecting log files and environment data
-Issue type: info
-Issue link: https://github.com/eclipse/codewind/issues/2766
-Info added in 0.12.0.
--->
-# Collecting log files and environment data
+## Build succeeded but project is stuck in the Stopped state
```

Comment:
This is more general troubleshooting than vscode-specific

Reply:
This is a very old Ts issue. Searched the Codewind issues and found only two that were both closed, so have removed this. The issue is covered by the `## Projects stuck in starting or stopped state` issue in the main troubleshooting section.